src/pages/ContentScripts/features/repo-sidebar-labels/view.tsx

```diff
@@ -0,0 +1,53 @@
+import getGithubTheme from '../../../../helpers/get-github-theme';
+import optionsStorage, { HypercrxOptions, defaults } from '../../../../options-storage';
+import '../../../ContentScripts/index.scss';
+import React, { useState, useEffect } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import '../../../../helpers/i18n';
+const githubTheme = getGithubTheme();
+
+type Label = {
+  id: string;
+  name: string;
+  type: string;
+};
+
+interface Props {
+  labels: Label[];
+}
+
+const View = ({ labels }: Props): JSX.Element | null => {
+  const [options, setOptions] = useState<HypercrxOptions>(defaults);
+  const { t, i18n } = useTranslation();
+  useEffect(() => {
+    (async function () {
+      setOptions(await optionsStorage.getAll());
+      i18n.changeLanguage(options.locale);
+    })();
+  }, [options.locale]);
+
+  const meta_labels = labels.map((label) => (
+    <span
+      key={label.id} // use label.id as the key
+      id={label.id}
+      className="topic-tag topic-tag-link"
+    >
+      {label.name}
+    </span>
+  ));
```

Comment:
@Xsy41 Great jobs!

a small tip: lowerCamelCase is usually adopted by JavaScript/TypeScript projects, so `metaLabels` is preferred here compared to `meta_labels` :)

Reply:
Thanks for your advice! 😄 I will immediately change to `metaLabels` and follow the JavaScript/TypeScript style ~ Thanks for pointing out this little detail! 🎉 @tyn1998